core_chat_data/src/commonMain/kotlin/net/thechance/mena/core_chat/data/contacts/source/remote/dto/ContactToAddDto.kt

```diff
@@ -0,0 +1,6 @@
+package net.thechance.mena.core_chat.data.contacts.source.remote.dto
+
+data class ContactToAddDto (
```

Comment:
```suggestion
data class ContactCreationRequestDto (
```
What about this name?